locales-pending/premium.ftl

```diff
@@ -351,3 +351,31 @@ high-risk-breach-none-sub-description-bank-account = Bank account info
 high-risk-breach-none-sub-description-cc-number = Credit card numbers
 high-risk-breach-none-sub-description-pin = PINs
 high-risk-breach-none-continue = Continue
+
+# Security recommendations
+
+security-recommendation-steps-title = Here’s our advice:
+security-recommendation-steps-cta-label = Got it!
+
+# Phone security recommendation
+
+security-recommendation-phone-title = Protect your email address
+security-recommendation-phone-description = Unfortunately you can’t take it back. But there are steps you can take to make sure you stay safe.
+security-recommendation-phone-step-one = Block spam numbers to prevent more junk calls
+security-recommendation-phone-step-two = Don’t click on links in texts from unknown senders; if it appears to be from a trusted source, call directly to confirm
+security-recommendation-phone-step-three = Use a <link_to_info>{ -brand-relay } phone mask</link_to_info> to protect your phone in the future
+
+# Email security recommendation
+
+security-recommendation-email-title = Protect your email address
+security-recommendation-email-description = Unfortunately you can’t fix this. But there are steps you can take to protect yourself.
+security-recommendation-email-step-one = Don’t click on links in emails from unknown senders; If it appears to be from trusted source, call directly to confirm
```

Comment:
This feels off. I'd expect something like visit the website or send an email, not call?

Reply:
Question is pending with content …

Reply:
We [received an answer](https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40?node-id=448:23643&mode=design#551650698) @flodolo — the string is supposed to stay as is.